badger_api/account.py

```diff
@@ -22,10 +22,10 @@ def fetch_all_claimable_balances(chain: str):
 
     """
     results = {}
-    total_pages = fetch_total_claimable_pages()
+    total_pages = fetch_total_claimable_pages(chain)
```

Comment:
can we write a test for fetch_all_claimable_balances function? some sanity check to make sure the range change is working as intended